utils/clients/tools/tests/test_isisicat_prefix_mapping.py

```diff
@@ -0,0 +1,87 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Test ICAT prefix mapping
+"""
+import unittest
+from unittest.mock import patch
+
+from utils.clients.tools.isisicat_prefix_mapping import fetch_instrument_fullname_mapping
+
+
+# pylint:disable=no-self-use,too-few-public-methods,too-many-public-methods
+class MockInstrumentQueryResult:
+    """
+    Mocks result of isisicat_prefix_mapping.client.execute_query for an instrument
+    """
+
+    def __init__(self, name, full_name):
+        self.name = name
+        self.fullName = full_name  # pylint:disable=invalid-name
```

Comment:
If you change this to self.fullname, you should be able to remove the # pylint:disable=invalid-name

Reply:
The reason for using camel case is because I'm mocking the result of execute query which returns a list of objects where fullName is an attribute.